inferno-framework/inferno-core
# FI-3395 Fix suite describe with routes

# Summary

Fix the bug where `bundle exec inferno suite describe <suite id>` command fails in test kits that mount custom routes. The bug was because the suites booter (i.e: `Inferno::Application.start('suites')`) needs lib/inferno.rb loaded, and the CLI wasn't loading it. The backdraw is that this kicks off loading the rest of Inferno, but I think the CLI is fast enough that this is negligible.

# Testing Guidance

1. Checkout this branch
2. In a sibling directory, clone your favorite test kit that mounts routes, say smart-app-launch-test-kit. 
3. Modify the test kit's Gemfile to use the checked out inferno core:
```
gem 'inferno_core', path: '../inferno-core'
```
4. `bundle install`
5. Run `bundle exec inferno suites` and pick any suite id
6. Run `bundle exec inferno suite describe <suite id>` and confirm it works
